app/forms/TelephoneNumberFormProvider.scala

```diff
@@ -23,12 +23,12 @@ import play.api.data.Form
 
 class TelephoneNumberFormProvider @Inject() extends Mappings {
 
-  def apply(): Form[String] =
+  def apply(messagePrefix: String): Form[String] =
     Form(
-      "value" -> text("telephoneNumber.error.required")
+      "value" -> text(s"$messagePrefix.error.required")
         .verifying(
           firstError(
-            isNotEmpty("value", "telephoneNumber.error.required"),
+            isNotEmpty("value", s"$messagePrefix.error.required"),
             regexp(Validation.telephoneRegex, "telephoneNumber.error.invalid.characters")
```

Comment:
@jassalrichy  is messagePrefix required here?

Reply:
It depends if the error message is specific to the entity.

Reply:
@manishsw required as error message is specific